kerastuner/engine/hyperparameters.py

```diff
@@ -74,6 +74,9 @@ def __init__(self, name, values, default=None):
                 'The default value should be one of the choices. '
                 'You passed: values=%s, default=%s' % (values, default))
 
+    def __repr__(self):
+        return f"Choice(name: {self.name!r}, values: {self.values}"
```

Comment:
Use `'` as the quote character for consistency.

Also display the value for `self.default`.